tests/snapshot_tests/snapshot_apps/hot_reloading_app_with_screen_css.tcss

```diff
@@ -0,0 +1 @@
+/* This file is purposefully empty. */
```

Comment:
Point of order: it isn't empty.

Reply:
How about "This file is purposefully devoid of TCSS rules."?